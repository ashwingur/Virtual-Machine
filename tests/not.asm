FUNC LABEL 0
    MOV REG 0 VAL 0
    MOV REG 1 VAL 255
    PRINT REG 0
    PRINT REG 1
    NOT REG 0
    NOT REG 1
    PRINT REG 0
    PRINT REG 1
    MOV REG 2 VAL 133
    NOT REG 2
    PRINT REG 2
    NOT REG 2
    PRINT REG 2
    RET