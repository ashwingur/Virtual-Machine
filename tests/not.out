0
255
255
0
122
133
